plot.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+from matplotlib.ticker import MultipleLocator
+
+plt.style.use('ggplot')
```

Comment:
Remove this. We should jsut use the default style.

Reply:
The journal requires ggplot style. We can't change this